### DISCUSSION TITLE
im interested learn from this repository, but i cannot search information for register to get LK_CLIENT_ID  and LK_CLIENT_SECRET

### DISCUSSION THREAD

**Discussion by feelfireff1** [0niel/university-app#329] [Q&A] [answered]
im interested learn from this repository, but i cannot search information for register to get LK_CLIENT_ID  and LK_CLIENT_SECRET, can non russian get it ?

**Comment by MintyRoma** [comment, marked as answer]
As I know that tokens used for OAuth Server of university and can't be presented for everyone. Only close range of developers may get that tokens. I think you may ignore that environment vars.
